Added some additional descriptions around Tenant config related properties

dvargaslantana opened the pull request:
Hey @seanmcilvenna, 

Here's an initial PR for the initial descriptions of tenant config related settings. One note, listId on line 1296 has a TODO. Any suggestions on how to best describe that field?

Thanks,
Daniel

seanmcilvenna commented:
Dropped the file into [Swagger Editor](https://editor.swagger.io/) and it reported some issues:
![image](https://github.com/lantanagroup/link/assets/5384694/181ee0c8-4b71-4cec-bf0d-60ab148d0434)
Please resolve the highlighted issues. The first two issues are known and OK to exist.

dvargaslantana commented:
> Dropped the file into [Swagger Editor](https://editor.swagger.io/) and it reported some issues: ![image](https://user-images.githubusercontent.com/5384694/247696558-181ee0c8-4b71-4cec-bf0d-60ab148d0434.png) Please resolve the highlighted issues. The first two issues are known and OK to exist.

Oh interesting, I was using https://editor-next.swagger.io/ to edit through the doc. But I'll use the one you linked and see what the issue may be.

dvargaslantana commented:
Hey @seanmcilvenna ,

I converted the unsupported 'examples' parameters to 'example'. Also, the Semantic error looked to have already existed prior to the pull request. From what I can tell, it's a duplicate incomplete bulk request, so I just removed it.

Thanks,
Daniel
